troubleshooting/missing-image-asset-causes-ios-publish-to-fail.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Error Publishing App to the iOS AppStore
+description: Publishing Application to the AppStore is Failing due to Missing Image Assets
+type: troubleshooting
+page_title: AppStore Publish Fails due to Missing Image Assets
+slug: invalid-image-asset-fail-ios-publish
+position:
+tags: appbuilder, publish
+teampulseid:
+ticketid: 1044213, 1044122
+pitsid:
+
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Code (AppBuilder)</td>	
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Publish - AppStore</td>	
+  </tr>
+  <tr>
+    <td>Mobile OS</td>
+    <td>iOS</td>	
+  </tr>
+</table>
+
+## Description
+Publishing an app to the iOS AppStore results in an error stating `ERROR ITMS-90032: Invalid Image Path`.
```

Comment:
Try to re-work this into a you-oriented statement. For example: When you attempt to publish your app to the Apple App Store, the following error message appears: `ERROR ITMS-90032: Invalid Image Path`.